frontend/app/(general)/page.tsx

```
@@ -5,6 +5,7 @@ import { getCurrentUser } from "@/lib/auth/session";
 import { GradientBackground } from "@/components/gradient-bg";
 import { FunFact } from "@/components/droplets/fun-fact";
 import { getRandomFunFactDroplet } from "@/lib/requests/droplet";
+import Wave from "react-wavify";
```

ESLint

[error] 'Wave' is defined but never used.

(@typescript-eslint/no-unused-vars)